ci-operator/step-registry/openshift/e2e/azure/csi/openshift-e2e-azure-csi-workflow.yaml

```diff
@@ -4,10 +4,7 @@ workflow:
     pre:
     - chain: ipi-azure-pre
     - ref: storage-conf-csi-azure-disk
-    - ref: storage-conf-feature-gate
     - ref: storage-conf-wait-for-csi-driver
```

Comment:
storage-conf-wait-for-csi-driver can be removed too.

Reply:
done